tests/test_step_sim.py

```diff
@@ -1,9 +1,10 @@
 import argparse
 import os
-from gymfc.envs.fc_env import FlightControlEnv 
```

Comment:
No actual changes appear in this file, if thats correct please remove.

Reply:
Looks like the import was missing, does this work for you without?